client/src/i18n/en/form.json

```diff
@@ -93,7 +93,7 @@
             "GENERIC": "The form contains errors.",
             "HAS_ERROR": "Contains an error",
             "HAS_ERRORS": "The form contains errors.  Please correct them and resubmit.",
-            "LOT_QUANTITY": "Lot Quantity",
+            "LOT_QUANTITY": "Batch number Quantity",
```

Comment:
'Batch' means `Lot`, so you don't need to add 'number' because 'Batch number' means 'Numero de lot'.

The good translation of `Quantité du lot` is 'Batch Quantity' or 'Lot Quantity'